int32 delay_start
---